src/Npgsql/NpgsqlTypes/PostgisTypes.cs

```diff
@@ -0,0 +1,726 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Npgsql;
+
+namespace NpgsqlTypes
+{
+#pragma  warning disable 1591
+        /// <summary>
+        /// Represents the identifier of the Well Known Binary representation of a geographical feature specified by the OGC.
+        /// http://portal.opengeospatial.org/files/?artifact_id=13227 Chapter 6.3.2.7 
+        /// </summary>
+        enum OgrIdentifier : uint
+        {
+            Point = 1,
+            LineString = 2,
+            Polygon = 3,
+            MultiPoint = 4,
+            MultiLineString = 5,
+            MultiPolygon = 6,
+            GeometryCollection = 7
+        }
+
+        /// <summary>
+        /// The modifiers used by postgis to extend the geomtry's binary representation
+        /// </summary>
+        [Flags]
+        enum OgrModifier : uint
+        {
+            HasSRID = 0x20000000,
+            HasMDim = 0x40000000,
+            HasZDim = 0x80000000
+        }
+
+        /// <summary>
+        /// A structure representing a 2D double precision floating point coordinate;
+        /// </summary>
+        public struct BBPoint
```

Comment:
I'm probably just being ignorant about PostGIS/OpenGIS, but still - how does this differ from `PostgisPoint`, does it make sense to use `PostgisPoint` for everything?